services/core-web/common/reducers/staticContentReducer.js

```diff
@@ -225,11 +227,11 @@ export const getNoticeOfWorkApplicationPermitTypeOptions = (state) =>
 export const getNoticeOfWorkApplicationReviewOptions = (state) =>
   state[STATIC_CONTENT].noticeOfWorkApplicationReviewOptions;
 
-const isStaticContentLoaded = (state) =>
-  Object.keys(state)
-    // eslint-disable-next-line no-prototype-builtins
-    .filter((p) => state.hasOwnProperty(p) && Array.isArray(state[p]))
-    .every((p) => state[p].length > 0);
+const isStaticContentLoaded = (state) => true;
+// Object.keys(state)
+//   // eslint-disable-next-line no-prototype-builtins
+//   .filter((p) => state.hasOwnProperty(p) && Array.isArray(state[p]))
+//   .every((p) => state[p].length > 0);
```

Comment:
Is this commented-out code needed?

Reply:
this is more to demonstrate the change, it would need to be revisited if we want to fuly adopt this change.

Reply:
Actually, i think this check is still the correct check, (make sure all keys have values of arrays with length greater than 0... let me check.